kirin_lex.py

```diff
@@ -4,4 +4,204 @@
 # Angel Seiji Morimoto Burgos	| A01281380
 
 
-import ply.lex as lex
\ No newline at end of file
+import ply.lex as lex
+
+tokens = (
+	'CLASS',
+	'INHERITS',
+	'CONSTANT',
+	'PUBLIC',
+	'PRIVATE',
+	'INDEPENDENT',
+	'CONSTRUCTOR',
+	'VAR',
+	'FUNC',
+	'MAT',
+	'VEC',
+	'NEW',
+	'IF',
+	'ELSE',
+	'FOR',
+	'WHILE',
+	'PRINT',
+	'SCAN',
+	'RETURN',
+	'AND',
+	'OR',
+	'XOR',
+	'LESS_EQUAL_THAN',
+	'GREATER_EQUAL_THAN',
+	'EQUAL',
+	'NOT_EQUAL',
+	'IMPORT',
+	'INT'
+	'DOUBLE',
+	'CHAR',
+	'BOOL',
+	'ID',
+	'CONST_I',
+	'CONST_F',
+	'CONST_STRING',
+	'CONST_CHAR',
+	'CONST_BOOL'
+	)
+
+literals = ['=', '+', '-', '*', '/', '>', '<', '~', '_', ',']
+ignore = ' \t\n'
```

Comment:
I think this should be **t_ignore**, since it is a special rule reserved by lex.py